### ISSUE TITLE
[release] 6.3.0 is ready for QA

### ISSUE THREAD

**Issue by ckprice** [mozilla-services/screenshots#2681]
@SoftVision-PaulOiegas could we please QA https://github.com/mozilla-services/screenshots/releases/tag/6.3.0 on stage?

**Comment by SoftVision-PaulOiegas** [comment]
Hi @ckprice , we can look over it but I don't think we can finalize testing today. Cosmin is in PTO and will return only on Tuesday as I've mentioned yesterday in the email for Screenshots team. Also the entire team will be out of office in 17th of April (Easter Monday).

I have assigned @SoftVision-EmilPasca to look over the build and at least confirm the fixes performed. But for sure Cosmin will run a full set of tests on Tuesday to be sure that we aren't missing anything major. 

If you have any concerns about this plan, please let me know.

**Comment by ckprice** [comment]
Thanks Paul. There are no major features/fixes landing with this release, so Tuesday should be fine.

**Comment by ckprice** [comment]
@SoftVision-PaulOiegas just a note that the add-on landed in today's Nightly (pref'd off). It would be great if we could get a sanity check that all is well there.

**Comment by SoftVision-CosminMuntean** [comment]
Hi guys,

Since the Monday Nightly could not be tested, we decided for the morning to test the Screenshots (6.3.0 version) on stage server using a custom repository build.
After the Tuesday Nightly build with the fix for bug [1357137](https://bugzilla.mozilla.org/show_bug.cgi?id=1357137) was released (around 14:30 our time), we ran our Full Functional suite on it.

The results are as follow:
Two new issues were found that are reproducible on stage custom build and also on current Screenshots version from latest Nightly build:
#2699 - Clicking the "Terms" and "Privacy Notice" links from Onboarding tour, open a new blank tab 
#2698 - After dragging the scrollbar, a selection is automatically performed when hovering the page

Other than that, no other issues found at the moment. However, I will continue performing some exploratory testing tomorrow on the Nightly build, just to be sure that I haven't missed anything.

**Comment by ckprice** [comment]
> After the Tuesday Nightly build with the fix for bug 1357137 was released (around 14:30 our time), we ran our Full Functional suite on it.
> ...
> Other than that, no other issues found at the moment. However, I will continue performing some exploratory testing tomorrow on the Nightly build, just to be sure that I haven't missed anything.

Great! We plan on requesting uplift to Beta tomorrow unless your testing comes up with anything major. We have your two [`gamma blockers`](https://github.com/mozilla-services/screenshots/labels/gamma%20blocker) triaged, and will request another uplift for these bugs on April 28th.

**Comment by SoftVision-CosminMuntean** [comment]
I ran another round of testing today on the latest Nightly build and I found a couple of issues.

New issues:
#2707 - A Screenshots error is displayed, if you drag a tab back in the first browser window
#2708 - The Screenshots button is not grayed out when it is inactive (Windows and Linux)

Potential new issues:
The [#2599](https://github.com/mozilla-services/screenshots/issues/2599#issuecomment-295278025) issue was fixed, but I have noticed a new strange behaviour probably caused by the fix. 
If a selection with reduced width is performed near the left part of the window, the "Cancel", "Download" and "Save" buttons are not entirely visible. 
I added a comment on the issue. Please let me know if I should log a new issue for this behavior. 

Despite the issues found, there are no major blockers. However, from my point of view, it will be great if the uplift from April 28th would also contain the fixes for #2708 and for the behavior observed after #2599 fix (if its considered a separate issue).

**Comment by SoftVision-CosminMuntean** [comment]
@ckprice is there a chance that the issue #2683 can be fixed until Screenshots will be uplifted in Beta. Or at lest until the second uplift on April 28th?
